<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>864</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(238,238,238)</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="transfButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;
font-size: 20px
</string>
    </property>
    <property name="text">
     <string>H (S)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="spiceButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>SPICE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="medButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>MEDICIÓN</string>
    </property>
   </widget>
   <widget class="GraphWidget" name="graphwidget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>761</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color:black;</string>
    </property>
   </widget>
   <widget class="GraphWidget" name="phaseGraph" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>761</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>BORRAR GRÁFICOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 75 italic 22pt &quot;Tw Cen MT&quot;;
</string>
    </property>
    <property name="text">
     <string>PLOT TOOL GRUPO 2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="xTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="yTextEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="changeModuleLabels">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Módulo -&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changePhaseLabels">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Fase -&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="transferenceCheck">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="spiceCheck">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="medCheck">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header location="global">graphwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
